### PR TITLE
chart: add important parameters to tikv and tidb config

### PR DESCRIPTION
**Pull request by weekface**
### What problem does this PR solve? 
- add important parameters to tikv and tidb config
- add move `status-addr` to start script

### Check List 

Tests 

Code changes

 - Has Helm charts change

Side effects

Related changes

 - Need to cherry-pick to the release branch

### Does this PR introduce a user-facing change?:
 
 ```release-note
NONE
 ```

### CODE REVIEW

File: charts/tidb-cluster/values.yaml [hunk 1]
```diff
@@ -161,53 +161,18 @@ tikv:
...
+  # # Here are some parameters you MUST to customize (Please configure in the above `tikv.config` section):
   # [storage.block-cache]
-  # ## Size of the shared block cache. Normally it should be tuned to 30%-50% of container's total memory.
-  # # capacity = "1GB"
-  # [raftstore]
-  # ## true (default value) for high reliability, this can prevent data loss when power failure.
-  # # sync-log = true
-  # # apply-pool-size = 2
-  # # store-pool-size = 2
+  #   shared = true
```
Comment (DanielZhangQD) [inline charts/tidb-cluster/values.yaml]: What if users deploy TiDB cluster of v2.x with this chart?

File: charts/tidb-cluster/values.yaml [hunk 2]
```diff
@@ -258,7 +223,11 @@ tidb:
   config: |
     [log]
     level = "info"
-  
+  # # Here are some parameters you MUST to customize (Please configure in the above 'tidb.config' section):
```
Comment (DanielZhangQD) [inline charts/tidb-cluster/values.yaml]: "MUST customize"

**Comment by weekface** [comment]
@aylei @DanielZhangQD PTAL again.

**Pull request merged**